### DISCUSSION TITLE
WebAssembly for C# projects

### DISCUSSION THREAD

**Discussion by ilievmark** [godotengine/godot-proposals#10310] [Platforms]
Hello

Is it possible to get Godot 4 build with web platform exporting feature for C# projects now, with what it have? (test access)
We would also help you to test the 2d project exporting and report with issues that we found from our side

We have project ongoing which will include web platform releases. And we dont want to downgrade to Godot 3 only for that...

Correct me if Im wrong, but so far there are issues with sounds only, no?

Or maybe it is possible to get the access to such Godot 4 builds + templates individually?

Thank you!

**Comment by AThousandShips** [comment]
For web support for C# see:
* https://github.com/godotengine/godot/issues/70796
